packages/io/src/controller.ts

```diff
@@ -132,7 +131,6 @@ export class BaseController {
 
         return route.middleware(ctx, next)
       } else {
-        ctx.err(new NotFound())
```

Comment:
When there are some controller routers, it will throw an error when the matching router is in the next controller. We'd better throw not found exception after all routers have matched.